pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -274,6 +293,11 @@ func parseQuery(model *simplejson.Json) (*CloudWatchQuery, error) {
 		alias = "{{metric}}_{{stat}}"
 	}
 
+	highResolution, err := model.Get("highResolution").Bool()
```

Comment:
When I test this PR I get an error. 

This should be `.MustBool(false)`

Reply:
Thanks for check!
Fix it.